### ISSUE TITLE
Icon-only destructive actions lack contextual accessible naming

### ISSUE THREAD

**Issue by RobertAgterhuis** [RobertAgterhuis/myAgentic-IT-Project-team-V2#1559]
Implements #1557

**Comment by RobertAgterhuis** [comment]
Closing duplicate created during UX backlog import; canonical issue retained.
